content/en/docs/install-operate/deployment.md

```diff
@@ -1,13 +1,23 @@
 ---
 title: Deployment
-description: Installing Falco on a Cluster
+description: Installing Falco on a Kubernetes Cluster
 aliases: [/docs/getting-started/deployment]
-weight: 40
+weight: 45
 ---
 
+The guides - [Download](/docs/install-operate/download/), [Install](/docs/install-operate/installation/), [Running](/docs/install-operate/running/) - focused on the systemd service deployment method, detailing how to primarily install and run Falco directly on the host. Additionally, these guides explored the alternative options of installing and running Falco within a container, with a driver installed on the underlying host.
+
+The container deployment option is particularly enticing for Kubernetes settings, which will be discussed in more detail now. 
+
+{{% pageinfo color="info" %}}
+
+Falco only needs to run once per Linux operating system, irrespective of the deployment strategy, whether on a bare-metal server or a virtual machine (VM). It takes advantage of the shared kernel in containerized infrastructures like Kubernetes through the installation of a kernel driver on the underlying host. Thus, deploying Falco within a container with the driver installed on the host is equivalent to deploying it directly on the Linux host. This allows seamless monitoring of both host and container workloads *without* the need for sidecars.
```

Comment:
```suggestion
Falco is designed to run once per Linux operating system, regardless of whether the deployment is on a bare-metal server or a virtual machine (VM). It leverages the shared kernel in containerized environments like Kubernetes by installing a kernel driver on the underlying host. This means that deploying Falco within a container, with the kernel driver installed on the host, is functionally equivalent to deploying it directly on the Linux host. This setup allows seamless monitoring of both host and container workloads *without* the need for sidecars.
```

Reply:
I think it may be the "Once per Linux operating system" that may be tripping us up. You can conceivably have several Linux operating systems running (via VMs), so it may really be that Falco is designed to run once per instance of the kernel?